labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
Your indentation is all over the place!

In your editor, try Ctrl+shift+p and then type 'indent' to search for a command that'll automatically re-indent the whole file (you may need to select all, first. If you can't figure it out, bug my sometime tomorrow)